server/lib/queries.js

```diff
@@ -64,14 +68,18 @@ module.exports = function(sequelize) {
       minTotalDonationClause = ''
     }
 
+    if (tag) {
+      tagClause = 'g.tags && $tag AND';
+    }
+
     return sequelize.query(`
       WITH "totalDonations" AS (
         SELECT "GroupId", SUM(amount) as "totalDonations", MAX(currency) as currency, COUNT(DISTINCT "GroupId") as collectives FROM "Transactions" WHERE amount > 0 AND currency='USD' AND "PaymentMethodId" IS NOT NULL GROUP BY "GroupId"
       )
       SELECT g.id, g.name, g.slug, g.mission, g.logo, g."backgroundImage", t."totalDonations", t.currency, t.collectives
       FROM "Groups" g LEFT JOIN "totalDonations" t ON t."GroupId" = g.id
-      WHERE ${minTotalDonationClause} g.tags && $tag AND g."deletedAt" IS NULL ${excludeClause}
-      ORDER ${orderClause} DESC NULLS LAST LIMIT ${limit}
+      WHERE ${minTotalDonationClause} ${tagClause} g."deletedAt" IS NULL ${excludeClause}
+      ORDER ${orderClause} ${orderDirection} NULLS LAST LIMIT ${limit} OFFSET ${offset || 0}
```

Comment:
Similar comment for `orderBy` and `orderDirection`, we already have `mw.sorting()`